src/test/java/com/dawn/jat/illuminati/post/controller/PostControllerTest.java

```diff
@@ -55,16 +55,16 @@ public static void init() {
     public void findBySlug_whenPostIdIsAvail_thenRetrievedPostIsCorrect() {
         String mockSlug = postEntity1.getSlug();
         Optional<PostEntity> mockPostEntities = Optional.of(postEntity1);
-
```

Comment:
Do we need any changes here, revert the file if there nothing change :-?

Reply:
Updated

Reply:
Reverted file